regexes/client/browsers.yml

```
@@ -1166,6 +1166,11 @@
   version: '$1'
   engine:
     default: 'Blink'
+- regex: 'Opera%20Touch/(\d+[\.\d]+)? CFNetwork/.+Darwin/.+(?!.*x86_64)'
```

Shouldn't this be enough? Or do you match the other parts for the Engine?

```suggestion
- regex: 'Opera%20Touch/(\d+[\.\d]+)?'
```

---

this rule is only for iOS mobile devices, where the browser engine is always webkit

---

ok 👍